man/man3/pmwebapi.3

```diff
@@ -175,6 +175,10 @@ parameter.
 The value passed in the request will be sent back in the
 response \- all responses will be in JSON object form in
 this case, with top level "client" and "result" fields.
+.PP
+REST API clients can optionally submit an URL-encoded query string
+in the body of the HTTP request unless otherwise noted. In this
```

Comment:
Marko likes new sentences on a new line.  :)

Reply:
In general, me too - usually I also ignore the general 80 character limit per line guidance, because I find it annoying if a diff produces a two-line-change only because I changed one sentence and had to break the sentence at a different position than before.

I've updated the commit with the line break.